test/controllers/declaration/RepresentativeDetailsPageControllerSpec.scala

```diff
@@ -158,7 +162,17 @@ class RepresentativeDetailsPageControllerSpec extends CustomExportsBaseSpec with
       header.headers.get("Location") must be(Some("/customs-declare-exports/declaration/additional-actors"))
     }
 
-    "accept form with status and address only" in {
+    "accept form with status and EORI if on standard journey" in {
```

Comment:
Here you can use the format:
`"accept form with status and EORI" when {
    "on standard journey" in {...}
    "on supplementary journey" in {...}
}`